pipelines/services/resource.go

```diff
@@ -0,0 +1,45 @@
+package services
+
+import (
+	"encoding/json"
+	"github.com/jfrog/jfrog-client-go/http/jfroghttpclient"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/jfrog/jfrog-client-go/utils/io/httputils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"net/http"
+)
+
+// GetPipelineResourceID fetches resource ID for given full repository name
```

Comment:
Let's explain as part of this description the returned values of this function.
In addition, let's use named returned values. When there are more than two returned values, the code becomes easier to read this way. Let's also make sure to shrten the `return` statements in this function as a result of thsi change.